app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -316,20 +316,19 @@ protected function getMultilineFieldConfig($attributeCode, array $attributeConfi
     protected function getDefaultValue($attributeCode)
     {
         switch ($attributeCode) {
```

Comment:
Correction, `$this->directoryHelper->getDefaultCountry()` doesn't need a customer. Let me draft up a commit.